assets/images/logo/speed-dial/README.md

```diff
@@ -0,0 +1,13 @@
+# Speed Dials
+
+  These are custom speed dials made with [Vivaldi Speed Dial Generator](https://vivaldi-thumbnails.netlify.app/)
+  and using the official theme colors.
+
+  They can be used for your bookmarks to the Modding Guide and to vivaldi://inspect.
+
+Direct Links:
```

Comment:
Not really direct.

Reply:
Do you want me to make them download links?

Reply:
Either link to the `raw.` URLs (e.g. https://raw.githubusercontent.com/code3z/modding-vivaldi/logo/assets/images/logo/speed-dial/sd-blue.png, which will, however, be `tiosgz` and `master` after merge), or don’t call them direct (in which case I don’t see any point in having the list 🤷).